cmd/kubeadm/app/cmd/config.go

```diff
@@ -71,7 +71,7 @@ func newCmdConfig(out io.Writer) *cobra.Command {
 		// cobra will print usage information, but still exit cleanly.
 		// We want to return an error code in these cases so that the
 		// user knows that their command was invalid.
-		RunE: cmdutil.SubCmdRunE("config"),
+		PreRunE: cmdutil.SubCmdRunE("config"),
```

Comment:
as you can see from the comment above the idea is to return exit code > 0 if a command is executed without a sub command.

this has been a consistent kubeadm behavior and changing it is not ideal even if one might argue about it.
`kubeadm config` (on it's own returns) returns exit code > 0.

instead of changing all existing commands to print a help screen we should just change kubeadm alpha to follow the same rule. this would also mean retitling this PR.